### ISSUE TITLE
Add option to copy URL on double click

### ISSUE THREAD

**Issue by djhtml** [keepassxreboot/keepassxc#10732]
## Summary
Please add option to copy URL on double click, instead of opening it. (Similarly to username/password.)

## Context
I like this feature in the original KeePass, it's handy when you use more browsers (not just the default one).

**Comment by SdoggaMan** [comment]
I second this!

**Comment by phoerious** [comment]
Just press Ctrl+U. We are not changing the behaviour of double click.

**Comment by droidmonkey** [comment]
I agree we have a shortcut, but this request could be handled with an option that adjusts the behavior of the double click.

**Comment by djhtml** [comment]
Thank you Jonathan. Yes I wanted to request an option, instead of changing the default behavior.
My original title might was a little misleading.

So clarifying it: please add an option (check box) to menu, where it's possible the select the behavior of double click on URL. Like in the original KeePass: 
<img width="380" alt="image" src="https://github.com/keepassxreboot/keepassxc/assets/26629618/d35b0a9a-5880-440b-9d21-e15b00da1a2b">

Thank you.

**Comment by phoerious** [comment]
We have a ton of settings already. I prefer not to add more if there are already viable ways without one.

**Comment by SdoggaMan** [comment]
I think the number of settings you currently have is unrelated to features users might want to use. 

KeePassXC is a fork of KeePass. I see it reasonable to expect features from KeePass to be supported in KeePassXC; my uneducated expectation as a user--and as a selling point--is that it's "KeePass +", or, KeePass but even more cool features. 

For my part I don't like using shortcuts for copying things in KeePass. I double click anything that won't auto-fill, so it actually matches my workflow to double click the username, double click the password, and double click the URL. I mean - I often share things to others (for example, the IP address for the router) and my fastest process by far is to double click the entry in question and slap it into my message. Keyboard shortcuts just add room for fat-fingering. 

Again, aware this would not be everyone's workflow, but my point is that the option should be there for those of us, like me, who want to use it.

**Comment by droidmonkey** [comment]
We are NOT a fork of KeePass. We implement KeePass2 kdbx format, but share absolutely no code or feature parity with KeePass.

**Comment by SdoggaMan** [comment]
Understood, my mistake. Easy enough to make I suppose when sharing the name and being so feature inter-operable, but I apologise for my incorrectness. 

I still share the same opinion; the two are obviously interoperable and equate to each other quite literally by name, and I feel like a lot of people 'upgrade' from KeePass 2 to a KeePass fork or alternate development (a la KPXC) so it seems reasonable to me to expect people used to KP2 to carry workflows from it over to here.

**Comment by droidmonkey** [comment]
I prefer to keep this as a keyboard (and toolbar) shortcut
